shared/js/views.jsx

```diff
@@ -831,6 +927,30 @@ loop.shared.views = (function(_, mozL10n) {
     }
   });
 
+  var RemoteCursorView = React.createClass({
+    mixins: [React.addons.PureRenderMixin],
+
+    propTypes: {
+      remoteCursorOffset: React.PropTypes.object,
+      remoteCursorPosition: React.PropTypes.object
```

Comment:
Does `RemoteCursorView` need to know about an offset and position? Any reason not to just have the props be a calculated top and left position?